apps/verticalhome/js/stores/item.js

```diff
@@ -217,8 +218,10 @@
           self.gridOrder = configurator.getGrid();
           self.populate(cb);
         });
-        window.addEventListener('configuration-pin-app-ready', function onPinAppReady() {
-          window.removeEventListener('configuration-pin-app-ready', onPinAppReady);
+        window.addEventListener('configuration-pin-app-ready',
```

Comment:
Guys, let's keep it as is please. In current circumstances we should focus only on those problems which are really major or critical. I understand that there are many minor and cosmetic issues, but we should not scatter our resources among them. The most important thing for now is to have basic functionality landed towards Harman gaia branch as soon as possible.